src/components/portal/products/ProductList.js

```diff
@@ -94,67 +93,69 @@ class ProductList extends Component {
 
   render() {
     //updated this to get the product from the global state instead of the parent
-    const product = this.props.product.products[this.props.index];
```

Comment:
this was put in because block didn't work properly without the direct link to the reducer, did you check that still worked?